Pareto/Checks/System Integrity/SecureTerminal.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  SecureTerminal.swift
+//  Pareto Security
+//
+//  Created by Janez Troha on 15/07/2021.
+//
+
+import Foundation
+import os.log
+import Regex
+import Version
+
+class SecureTerminalCheck: ParetoCheck {
+    static let sharedInstance = SecureTerminalCheck()
+    override var UUID: String {
+        "5cbe1cfd-ff28-4cc7-8998-5d72e608b28d"
+    }
+    
+    override var TitleON: String {
+        "Other apps cannot read from Terminal.app"
+    }
+    
+    override var TitleOFF: String {
+        "Other apps can read from Terminal.app"
```

Comment:
"Other apps can detect and record what you type in Terminal."

Reply:
![image](https://user-images.githubusercontent.com/239513/169027164-33aede90-c278-44b3-966a-b3c1e4d0e935.png)

long titles are problematic, same goes for menubar app

Reply:
Does this fit?

"Other apps can't record typing in Terminal"?